features/doctrine/order_filter.feature

```diff
@@ -360,6 +360,66 @@ Feature: Order filter on collections
     }
     """
 
+  Scenario: Get collection ordered in ascending order on an embedded and on which order filter has been enabled in whitelist mode
+    Given there is "30" dummy objects with embeddedDummy
+    When I send a "GET" request to "/embedded_dummies?order[embeddedDummy]=asc"
```

Comment:
Just a question @jordscream  as I don't see test for that, would sending a request to `"/embedded_dummies?order[embeddedDummy.dummyName]=asc"`  work ?